### PR TITLE
fix: modify v16 upgrade handler

### PR DESCRIPTION
**Pull request by czarcas7ic**
Closes: #XXX

## What is the purpose of the change

The community pool now has the DAI it needs to create the first position, so the logic for swapping is now removed.

Additionally, since we send funds via the community pool, the community pool entry in feePool within the distribution module must also be manually set. This was discovered via state exported testnets.

### CODE REVIEW

**Comment by czarcas7ic** [comment]
need to update e2e to have it's community pool funded with 1dai for this to pass, but if you are reviewing this should still be good to ack

File: app/upgrades/v16/upgrades.go [hunk 1]
```diff
@@ -134,25 +135,38 @@ func CreateUpgradeHandler(
...
+		// Remove coins we used from the community pool to make the CL position
+		feePool := keepers.DistrKeeper.GetFeePool(ctx)
+		newPool, negative := feePool.CommunityPool.SafeSub(sdk.NewDecCoinsFromCoins(fullRangeOsmoDaiCoins...))
+		if negative {
+			return nil, fmt.Errorf("community pool cannot be negative: %s", newPool)
+		}
+
+		// Update and set the new fee pool
+		feePool.CommunityPool = newPool
+		keepers.DistrKeeper.SetFeePool(ctx, feePool)
```
Comment (p0mvn) [inline app/upgrades/v16/upgrades.go:168]: Could you please add tests to check that the community pool balance as well as this `feePool` are updated in the `upgrades_test.go`?

Comment (czarcas7ic) [reply app/upgrades/v16/upgrades.go]: Test added! https://github.com/osmosis-labs/osmosis/pull/5659/commits/4cece9a3c0b23c6d77995c72d1b6f06faadc2823

**Review by p0mvn — APPROVED** [review]

**Comment by p0mvn** [comment]
Nice work!

**Pull request merged**